src/libraries/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -44,6 +44,7 @@ public void RelativeSearchPath_Is_Null()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // throws pNSE
```

Comment:
This seems to be using `RemoteExecutor`. Exclude it using `[ConditionalFact(typeof(RemoteExecutor)...`?

Reply:
Hmm... My guess is that we're going to need a different test even w/ RemoteExecutor enabled. Process makes sense when running on node, but in a browser, not so much.